### DISCUSSION TITLE
Sources for web page https://tteck.github.io/Proxmox/

### DISCUSSION THREAD

**Discussion by gmacario** [tteck/Proxmox#2486] [Q&A]
First of all, many thanks @tteck for this wonderful teasure trove for Proxmox!

Back to my question... The title says it all.

I spotted a tiny typo under Server - Networking > Tailscale

![image](https://github.com/tteck/Proxmox/assets/75182/9a6254c0-b4a9-4cd3-afb2-1e5031f26c2b)

and wanted to submit a PR to fix it but could not figure out where exactly the sources of the website are located.

As far as I could understand branch https://github.com/tteck/Proxmox/tree/gh-pages is generated from some source tree,
but could not figure out where the sources are maintained.

P.S.: My apologies if the answer is trivial or already written somewhere.

**Comment by tteck** [comment]
Fixed, thanks

**Comment by gmacario** [comment]
> Fixed, thanks

You are welcome 😉 

Does that imply that the sources of the website <https://github.com/tteck/Proxmox/> are not currently publicly available?

**Comment by Nudelsalad** [comment]
Hi @tteck, I tried to find any information on how you actually design the website and stumbled across this discussion. 
I want to fork your repository and add my own scripts to the website but it is quite hard to modify per text. Which framework/tools do you use and how to set them up? A Readme.md in the root of the gh-pages branch would be awesome describing your setup. 
Thanks in advance @tteck 
Appreciate any answer

P.S.: @gmacario the build process can be viewed here https://github.com/tteck/Proxmox/actions/runs/8542377756/job/23403859110
A simple `bundle exec jekyll build`

**Reply by tteck** [reply]
The source of the website is not publicly available.
